xclim/data/fr.json

```
@@ -513,67 +512,67 @@
     "description": "Température moyenne du mois le plus chaud avec un facteur d'échelle basé sur la latitude  : Moyenne de température du mois le plus chaud * ({lat_factor} - latitude).",
     "title": "Indice de latitude-température",
     "comment": "Formule originale prise de Jackson, D. I., & Cherry, N. J. (1988)",
-    "abstract": "indice climatique basé sur la température moyenne du mois le plus chaud et un coefficient basé sur la latitude pour tenir compte de la durée plus longue du jour favorisant les conditions de croissance. Développé spécifiquement pour la viticulture."
+    "abstract": "Indice climatique basé sur la température moyenne du mois le plus chaud et un coefficient basé sur la latitude pour tenir compte de la durée plus longue du jour favorisant les conditions de croissance. Développé spécifiquement pour la viticulture."
   },
   "FRESHET_START": {
-    "long_name": "Jour de l'année du début de la crue nivale",
-    "description": "Jour de l'année du début de la crue nivale, défini comme le {window}e jour consécutif où la température excède {thresh}.",
-    "title": "Nième jour consécutif où la température excède un seuil",
+    "long_name": "Premier jour de dépassement constant du seuil de température",
+    "description": "Jour de l'année du début de la crue printanière, défini comme le premier jour où un seuil de température de {thresh} est dépassé pendant au moins {window} jours.",
+    "title": "Jour de l'année du début de la crue nivale",
     "abstract": "Premier jour où la température excède un certain seuil depuis un certain nombre de jours consécutifs."
   },
   "FROST_SEASON_LENGTH": {
-    "long_name": "Durée de la saison de gel (Tmin < 0°C)",
-    "description": "Durée de la saison de gel, définie comme la période où Tmin est sous 0°C sans redoux de {window} jours ou plus.",
+    "long_name": "Durée de la saison de gel",
+    "description": "Nombre {freq} de jours entre la première occurrence d'au moins {window} jours consécutifs avec une température minimale journalière inférieure au point de congélation et la première occurrence d'au moins {window} jours consécutifs avec une température minimale journalière supérieure au point de congélation après {mid_date}.",
     "title": "Durée de la saison de gel",
-    "abstract": "Durée de la saison de gel, définie comme la période où Tmin est sous 0°C sans redoux de {window} jours ou plus."
+    "abstract": "Durée de la saison de gel, définie comme la période pendant laquelle la température minimale quotidnne est inférieure à 0°C sans fenêtre de dégel de plusieurs jours, le dégel se produisant après une date calendaire médiane."
```

```suggestion
    "abstract": "Durée de la saison de gel, définie comme la période pendant laquelle la température minimale quotidienne est inférieure à 0°C sans fenêtre de dégel de plusieurs jours, le dégel se produisant après une date calendaire médiane."
```

Aussi, c'est quoi une date calendaire médiane? Est-ce qu'on peut juste dire "après une date donnée".
Si on le change ici, il faudrait changer toutes ses autres apparitions aussi.

---

"Date calendaire" c'est pour préciser que la date n'a pas d'information d'année. On le précise en anglais avec "calendar date". Cependant, l'alternative proposée ne me choquerait pas non plus.

---

Je note que personne ne semble utiliser cette expression...

---

Linguee montre qu'on dit parfois "date du calendrier", mais le plus souvent "date" tout court.

---

Woups, mon idée était "date du calendrier" aussi. Ce n'est probablement pas choquant que deepl propose des choses qui entendent un peu étrange.